react/components/ProductImages/components/Zoomable/ZoomInPlace.tsx

```diff
@@ -78,15 +87,20 @@ const ZoomInPlace: FC<Props> = ({ children, zoomContent, type, factor }) => {
     setZoom(!isZoomedIn)
 
     if (!isZoomedIn) {
-      const mousePosition = getMousePositionFromEvent(event, getContainerBounds())
+      const mousePosition = getMousePositionFromEvent(
+        event,
+        getContainerBounds()
+      )
       setPositionAndScale(mousePosition.x, mousePosition.y, factor)
     }
   }
 
   // Resets position when the image is zoomed out
   useEffect(() => {
     if (!isZoomedIn) {
-      setPositionAndScale(0, 0, 1)
+      // This value of the scale is because of some really weird bug
+      // in chrome https://github.com/vtex-apps/store-components/issues/738
+      setPositionAndScale(0, 0, 1.000001)
```

Comment:
```suggestion
      // This value of the scale is because of some really weird bug
      // in Chrome https://github.com/vtex-apps/store-components/issues/738
      // Should be reverted to just `1` once the Chrome bug is fixed.
      setPositionAndScale(0, 0, 1.000001)
```